models/agent_work_time/int_zendesk__ticket_work_time_business.sql

```diff
@@ -32,16 +32,16 @@ with ticket_historical_status as (
 
     select 
       ticket_status_crossed_with_schedule.*,
-      round({{ fivetran_utils.timestamp_diff(
```

Comment:
Postgres does not have a dynamic rounding function. In order to round with postgres you must explicitly convert the data type to numeric. Looking through most of the round functions within this package I felt it would cause more downstream casting issues if we explicitly cast all fields we round. 

As such, I opted to remove the rounding functions and can leave the rounding up to the end use within the BI layer. If we want to go the route of casting instead I can update all the round changes.